src/components/ProjectIndex.vue

```diff
@@ -1,26 +1,30 @@
 <template>
   <v-container class="fill-height">
     <v-responsive class="align-center fill-height">
-      <v-img height="150" src="https://github.com/Neoforged.png"/>
+      <v-img height="150" src="https://github.com/Neoforged.png" />
       <h1 class="text-center">NeoForged Project Index</h1>
 
-      <div class="py-6"/>
+      <div class="py-6" />
 
       <v-row class="d-flex align-center justify-center">
-        <v-col v-for="repo in Object.keys(data)" :key="repo" cols="12" sm="5">
-          <v-card class="mx-auto" :title="data[repo].name" :href="`/${repo}`">
+        <v-col v-for="repo in repos" :key="repo.projectRepo" cols="12" sm="5">
+          <v-card class="mx-auto" :title="repo.name" :href="`/${repo.projectRepo}`">
             <template v-slot:append>
               <Suspense>
-                <RepoChips :repo="repo" />
+                <RepoChips :repo="repo.projectRepo" />
               </Suspense>
             </template>
+
             <v-card-item>
               <Suspense>
-                <RepoDescription :repo="repo" />
+                <RepoDescription :repo="repo.projectRepo" />
               </Suspense>
             </v-card-item>
+
             <v-card-actions>
-              <v-btn :href="`/${repo}`">Details</v-btn>
+              <v-btn :href="`/${repo.projectRepo}`">Details</v-btn>
+              <v-spacer></v-spacer>
+              <v-btn :href="`/${repo.projectRepo}`">{{ repo.latestVersion || 'N/A' }}</v-btn>
```

Comment:
I don't like the N/A, just make it show nothing. Additionally, the use of a button is quite weird as it changes the font, it should be simple text.